expression/schema.go

```diff
@@ -259,3 +260,13 @@ func MergeSchema(lSchema, rSchema *Schema) *Schema {
 func NewSchema(cols ...*Column) *Schema {
 	return &Schema{Columns: cols, TblID2Handle: make(map[int64][]*Column)}
 }
+
+// FindIndexByName finds index by name.
```

Comment:
Why move it here?

Reply:
After talk with @zz-jason , we decided to move this function to here.